tests/conftest.py

```diff
@@ -31,12 +32,19 @@ def __init__(self, display, hlwm_process):
         # by self.hc_idle
         self.wmclass2winid = {}
 
-    def _checked_call(self, cmd, expect_success=True):
+    def _possibly_split_command(self, cmd):
```

Comment:
I found this name a bit confusing. I think somethink like `_prepare_cmd_args` (or "parse") would be better. (Don't say what it does, but say what it's good for.)